third_party/terraform/data_sources/data_source_google_firebase_web_app.go.erb

```diff
@@ -0,0 +1,36 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+import (
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceGoogleFirebaseWebApp() *schema.Resource {
```

Comment:
👍 thanks for adding the data source as well. Can you add a datasource test for this too?